### PR TITLE
add supported engine. Fix #45

### LINKED ISSUE #45: supported node version
which node version is this intended to follow? lts? I'm asking this because we can create a `.nvmrc` file at the root to help devs.

### PR DESCRIPTION
**Pull request by PlayMa256**
Keep or not keep it strict? (adding to engines field)

If the decision is yes, the container entria/node-ci has to be updated.

### CODE REVIEW

**Comment by miltoneiji** [comment]
Should we remove the .nvmrc that exists at the server package?

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+lts/*
```
Comment (sibelius) [inline .nvmrc]: is this right?

Comment (PlayMa256) [reply .nvmrc]: yes: https://github.com/creationix/nvm#nvmrc

**Comment by PlayMa256** [comment]
> Should we remove the .nvmrc that exists at the server package?

yes, good catch.

**Pull request merged**